src/lib/Scenes/Home/Components/Auctions/index.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react"
+import { FlatList, SectionList, TouchableWithoutFeedback, View } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components/native"
+
+import fonts from "../../../../../data/fonts"
+import { AuctionItem } from "./Components/AuctionItem"
+
+const Container = styled.View`
+  flex: 1;
+  padding: 10px 15px;
+`
+
+const SectionHeader = styled.View`
+  padding-top: 15px;
+  padding-bottom: 10px;
+  background-color: white;
+`
+
+const SectionTitle = styled.Text`
+  font-family: ${fonts["garamond-regular"]};
+  font-size: 25px;
+  text-align: left;
+  margin-left: 2px;
+`
+
+interface Props {
+  auctions: Array<{
+    live_start_at: string | null
+  }>
+}
+
+class Auctions extends React.Component<Props, null> {
+  handleTap(item) {
+    console.log(item)
```

Comment:
Should this do something (also, types?)

Reply:
log should go, but I like the separate method for analytics. It can be removed for this PR, I'll probs add it back in later on :D.

Reply:
Sounds good, I'm not sure how to get a url to route to the appropriate VC yet so I'll leave the method blank or figure it out now.